cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -105,7 +105,7 @@
             <div class="o-post-preview_image-container">
                 <img class="o-post-preview_image"
                      src="{{ photo.url }}"
-                     alt="{{ post.preview_image.alt if post.preview_image.alt else '' }}">
+                     alt="{{ photo.alt }}">
```

Comment:
@Scotchester is it nothing at all, or is it just the image rendition instance returned from

```html
{% set photo = image(post.preview_image, 'original') %}
```

I feel okay about modifying `image_alt_value` to support that as an alternate input type.